postgis/run_postgres

```diff
@@ -1,42 +1,33 @@
 #!/usr/bin/with-contenv sh
 
-mkdir -p ${PGDATA}
-
-# taken care of in initialization
-#chown -R postgres:postgres ${PGDATA}
-
-# if data folder is empty, initialize db
-test "$(ls -A "$PGDATA" 2>/dev/null)" || s6-setuidgid postgres /usr/bin/initdb -D $PGDATA
-
-# add md5 auth option for all internal clients
-[ $(grep -cE "host(\s)+all(\s)+all(\s)+0.0.0.0/0(\s)+md5" $PGDATA/pg_hba.conf) -eq 1 ] || \
-  echo "host all all 0.0.0.0/0 md5" >> $PGDATA/pg_hba.conf
-
-# add a user, pass and a database owned by that user
-TMP_FILE=/tmp/pgsql.initialize.txt
-# also add postgis extensions to that db
-TMP_GIS_FILE=/tmp/pggis.initialize.txt
-if [ ! -f $TMP_FILE ]; then
-    # create user and db
-    PGSQL_USER=${PGSQL_USER:-"dbuser"}
-    PGSQL_PASS=${PGSQL_PASS:-"dbpass"}
-    PGSQL_DB=${PGSQL_DB:-"dbname"}
-    cat > $TMP_FILE << EOSQL
+mkdir -p $PGDATA
+
+# if data folder is empty
+if [ "$(find $PGDATA ! -path $PGDATA | wc -l)" == "0" ]; then
```

Comment:
I suspect this is very slow if there are a lot of files under $PGDATA; find will try to list all of them before piping the output to `wc`. You should add a `-maxdepth` param or be able to use `ls -A $PGDATA | wc -l` instead. That will *not* list dotfiles, but if there is data, there should be a subdir that is not a dotfile.

Reply:
`ls -A` sucked for some reason. I'll add `-maxdepth`